Bottom: 425e513de1f281e01f09389fc04b2c1cfbbd6993
Top:    3acb3bf37d32e55a84ae17001bac0bc3582e94ad
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-01-18 16:14:29 +0000

Fix for PCB+GL Experimental


---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index ad816d3..1b0e79f 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -851,9 +851,9 @@ do_hole (const BoxType *b, void *cl)
 }
 
 static bool
-polygon_contains_user_holes (PolygonType *polygon)
+pour_contains_user_holes (PourType *pour)
 {
-  return (polygon->HoleIndexN > 0);
+  return (pour->HoleIndexN > 0);
 }
 
 
@@ -887,7 +887,7 @@ hidgl_fill_pcb_polygon (PolygonType *poly, const BoxType *clip_box /*, bool forc
   /* Polygon has holes.. does it have any user-drawn holes?
    * If so, it must be masked with a _new_ stencil bit.
    */
-  use_new_stencil = force_new_stencil | polygon_contains_user_holes (poly);
+  use_new_stencil = force_new_stencil || true; // | polygon_contains_user_holes (poly);
 
   if (use_new_stencil)
     {
